gbus/messages.go

```diff
@@ -18,7 +18,10 @@ type BusMessage struct {
 	Semantics         Semantics /*cmd or evt*/
 	Payload           Message
 	PayloadFQN        string
+	payloadBody       []byte
 	RPCID             string
+	Headers           amqp.Table
+	ContentType       string
```

Comment:
@hadascoh  Why do we need the ContentType on the BusMessage ?
The BusMessage is a higher-level abstraction over the amqp.Delivery and amqp.Publishing so intuitively I can't see the reason of placing this here. 
Can you please explain the use case you are trying to solve so we can find the best design to accommodate that use case ?

Reply:
I need to send an avro event from bp_risk service.
the event is registered in the event collector service and we want to send it through rabbit so the bp_event_publisher will handle it.
so I have 2 options:
1. use the avro serializer - but then I need to register also in grabbit - not scale for a lot of services and event ( and I didn't manage to do it - I think we missing something )
2. use the gob serializer - as I wrote before - its generic and we need to be able to send the right headers, contentType and payloadBody so the full flow: go_serveices -> grabbit -> bp_event_publisher -> event collector will work